ckanext/dia_theme/templates/user/dashboard_organizations.html

```diff
@@ -0,0 +1,25 @@
+{% extends "user/dashboard.html" %}
+
+{% block page_primary_action %}
+  {% if h.check_access('organization_create') %}
+    {% link_for h.humanize_entity_type('organization', org_type, 'add link') or _('Add Organization'), named_route=org_type ~ '.new', class_="btn btn-primary", icon="plus-square" %}
+  {% endif %}
+{% endblock %}
+
+{% block primary_content_inner %}
+  <h2 class="hide-heading">{{ h.humanize_entity_type('organization', org_type, 'my label') or _('My Organizations') }}</h2>
+  {% set organizations = h.organizations_available(permission='manage_group',
```

Comment:
permission should still be 'manage_group' or there is a org specific permission like `organization_create`?

Reply:
Good question, will check

Reply:
This is what the template looks like in CKAN 2.9.11: https://github.com/ckan/ckan/blob/ckan-2.9.11/ckan/templates/user/dashboard_organizations.html

2.10.x seems to have introduced the bug, and 2.9.11 still references `manage_group` as the perm here, so should be correct. Thanks for checking :)

Reply:
Also, seems someone has [tried to fix this issue in CKAN core](https://github.com/ckan/ckan/pull/7432), but it hasn't quite gone to plan... so we'll override for now and recheck at next upgrade to see if it's fixed.